### PR TITLE
JS best practices 

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we perfomed :

- Give feedback on someone else's project based on clean code best practices.
- Apply knowledge the student has gained in the past when giving a code review.
- Perform a simple code review for a coding partner.
- Understand existing code in a way that allows the student to give a good code review.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const inputBox = document.querySelector('.input-field');
+
+inputBox.addEventListener('keyup', (event) => {
+  if (event.keyCode == '13' && inputBox.value !== '') {
+    const myDescription = inputBox.value;
+    const NewTask = new TaskObj(myDescription);
+    inputBox.value = '';
+    myUiTask.addTask(NewTask);
+  }
+});
```
Comment (HamzaAnwar1998) [inline src/index.js:22]: I would suggest you use form submit event instead of using event.keyCode

**Review by HamzaAnwar1998 — APPROVED** [review]
Hi @AnicetFantomas 

Good job! I appreciate your effort in following JavaScript best practices. You can merge your branch

File: src/index.js (hunk 1 above)
Comment (AnicetFantomas) [inline src/index.js:22]: I would suggest to use a form submit instead of using keyCode. This will improve the user experience

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+- HTML
+- CSS
+- JavaScript
+- webpack
+
+## Live Demo is avaible here
+
+[Live Demo Link](https://anicetfantomas.github.io/To_do_list/)
+
+
```
Comment (AnicetFantomas) [inline README.md:23]: The live demo link does not contain all the expected functionalities, the user should be able to add a new task and clear all completed. Kindly fix it

**Pull request merged**